pkg/agent/api/delegatedidentity/v1/service.go

```diff
@@ -155,6 +176,19 @@ func sendX509SVIDResponse(update *cache.WorkloadUpdate, stream delegatedidentity
 		return err
 	}
 
+	log = log.WithField(telemetry.Count, len(resp.X509Svids))
+
+	// log details on each SVID
+	// a response has already been sent so nothing is
+	// blocked on this logic
+	for i, svid := range resp.X509Svids {
+		ttl := time.Until(update.Identities[i].SVID[0].NotAfter)
+		log.WithFields(logrus.Fields{
+			telemetry.SPIFFEID: svid.X509Svid.Id.String(),
+			telemetry.TTL:      ttl.Seconds(),
+		}).Debug("Fetched X.509 SVID")
```

Comment:
There is already a log statement in the `FetchX509SVIDs` Workload API handler with the same message. I think we should have a unique log message here to disambiguate where the log originated from.